packages/app/pages/index.tsx

```diff
@@ -376,7 +439,7 @@ const CreateMembershipModal = ({
       const weiPrice = web3.utils.toWei(price, "ether");
       return new Promise<void>((resolve, reject) =>
         contractInstance.methods
-          .create(name, symbol, quantity, weiPrice, metadataHash)
+          .create(name, symbol, quantity, weiPrice, metadataHash, claimable)
```

Comment:
You can pass royaltyPcnt as 0 for now when you rebase with royalties PR.  I will be pushing a PR for UI side royalty changes once this PR is merged.